### PR TITLE
fix update query var utility; add tests

### PR DESCRIPTION
**Pull request by GeoffDusome**
## What does this do/fix?

The `updateQueryVar` utility was not properly removing query vars set to `undefined`. This fix pulls in a version of `updateQueryVar` that adds all necessary functionality. This fix also includes tests for query vars that have an `undefined` value.

## Tests

Does this have tests?

- [x] Yes, tests have been updated accordingly to match new functionality. It's possible it need a few more though to test for null values as well. Let me know if we need those as well and I can get them added in.
- [ ] No, this doesn't need tests because...
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

File: wp-content/themes/core/assets/js/test/tests/utils/data/update-query-var.test.js [hunk 1]
```diff
@@ -30,4 +31,22 @@ describe( 'updateQueryVar', () => {
...
+	it( 'remove a key that has been set to undefined', () => {
+		const url = 'http://localhost/?param1=true&param2=1024';
+		const key = 'param1';
+		const value = undefined;
+		const expectedUrl = 'http://localhost/?param2=1024';
+
+		expect( updateQueryVar( key, value, url ) ).toBe( expectedUrl );
+	} );
+
+	it ( 'do not add a key that is set to undefined', () => {
```
[the remark this answers is not in the document]
Comment (GeoffDusome) [reply wp-content/themes/core/assets/js/test/tests/utils/data/update-query-var.test.js]: Removed space, should be resolved.

**Review by dpellenwood — APPROVED** [review]
This looks great! Thanks Geoff. One last ask, add [a changelog entry](https://github.com/moderntribe/square-one/blob/main/changelog.md), before you merge, if you would be so kind?
